editor/layouts/layout-tools.mdx

```diff
@@ -0,0 +1,93 @@
+
+After creating an Absolute Layout, you can drag it to reposition it, or resize it using the resize handles.
+
+<Tip>Absolute Layouts can be dragged around like other Rive objects. By default, if you drag a Layout (or any other object) inside the bounds of another Layout, it will display an indicator showing you that the dragged object will become a child of that Layout if dropped. Hold `command` / `control` while dragging the object to prevent existing items from being moved into the Layout.</Tip>
+
+  <video width="700" controls>
+    <source src="https://ucarecdn.com/994d9855-4783-478f-a808-e5f8e57511af/" type="video/mp4"></source>
+    Layout
+  </video>
+
+- **Row/Column:** The Row and Column tools create a Layout in the same way as the Layout tool above, but also include an initial set of children which will be positioned in either a Row or Column. You can use either the number keys or the up/down keys to define the number of children while dragging your row or column onto the stage. The created children will have their widths and heights set to **Fill**.
+
+  <video width="700" controls>
+    <source src="https://ucarecdn.com/e6fb26b0-a322-4715-8764-f484b99fd70a/" type="video/mp4"></source>
+    Row/Column
+  </video>
+
+---
+
+### Wrap in Layout
+
+Instead of starting with an empty Layout container, you can wrap existing objects into a Layout. There are a number of ways to wrap an active selection in a Layout:
+
+- Right click on the stage or the hierarchy to surface the context menu. Select `Wrap in` &gt; `Layout`. You can do this for a single or multiple objects at once.
+
+  <video width="700" controls>
+    <source src="https://ucarecdn.com/8dc09645-bb6b-4572-a519-5f655d39e114/" type="video/mp4"></source>
+    Wrap in
+  </video>
+
+- Use the `shift` + `L` shortcut.
+
+  <video width="700" controls>
+    <source src="https://ucarecdn.com/fc768d5a-33bf-4ce5-88fe-c40257263bc2/" type="video/mp4"></source>
+    `shift` + `L` shortcut
+  </video>
+
+- Alternatively, you can use the `Layout selection` button in the inspector. This is available when only non-Layout objects are selected.
+
+  <video width="700" controls>
+    <source src="https://ucarecdn.com/cca29ef7-5762-40a7-bfb2-0b1ac9da5ee8/" type="video/mp4"></source>
+    Layout selection
+  </video>
+
+---
+
+### Add Child Layout
+
+When a Layout is the current selection, an `Add Child Layout` button will appear in the Layout inspector. Clicking this will add a new Layout as a child of the selected Layout, with it's width and height set to Fill.
+
+![Image](/images/editor/add-child-layout.png)
+
+---
+
+### Dragging and Dropping Layouts
+
+Layouts (both Absolute and Relative) can be dragged and dropped into other Layouts at any time. This can be done in 2 ways:
+
+1. Drag and drop Layouts in the hierarchy panel.
```

Comment:
These should be bullet points. Typically you only use numbers for progression. Step 1, step 2, etc.